src/screens/CreateTokenAmount.js

```diff
@@ -114,7 +114,7 @@ class CreateTokenAmount extends React.Component {
         <HathorHeader
           title={t`CREATE TOKEN`}
           onBackPress={() => this.props.navigation.goBack()}
-          onCancel={() => this.props.navigation.pop()}
+          onCancel={() => this.props.navigation.getParent()?.goBack()}
```

Comment:
What happens if there is no parent?

Reply:
This `?` operator was put there mostly as a coding best practice, but the idea is that there will always be a parent navigator there.

Not having a parent is a signal that there was a major refactor and our current hierarchy of navigators is different from expected. In that case, maybe throwing an exception would be a better way of informing the developer of the need to change this code as well.

Reply:
Removed the _optional chaining_ operator on [a38a087](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/325/commits/a38a08757c5ed5741346ab6db2931fdc0b992d41) .